packages/devtools_app/lib/src/screens/network/network_screen.dart

```diff
@@ -47,7 +45,7 @@ class NetworkScreen extends Screen {
   @override
   Widget buildStatus(BuildContext context, TextTheme textTheme) {
     final networkController = Provider.of<NetworkController>(context);
-    final color = Theme.of(context).textTheme.bodyText2.color;
+    final color = Theme.of(context).textTheme.bodyText2!.color!;
```

Comment:
How about assigning default color instead of null checking?

Reply:
@kenzieschmoll may have a different opinion on this, but if these are null that means we've setup our theming incorrectly. These should never actually be null AFAIK.

Reply:
It is fine to null-assert. Especially when we know this was fine pre-migration.